server/driver_tidb_test.go

```diff
@@ -88,4 +88,26 @@ func (ts tidbResultSetTestSuite) TestConvertColumnInfo(c *C) {
 	}
 	colInfo = convertColumnInfo(&resultField)
 	c.Assert(colInfo, DeepEquals, createColumnByTypeAndLen(mysql.TypeTiny, 1))
+
```

Comment:
Is it possible to add an integration test?

Reply:
prepare/execute may be hard to test.

Reply:
OK, I see.